jenkins-design-language/src/js/components/Linkify.jsx

```diff
@@ -1,6 +1,7 @@
 // @flow
 
 import React from 'react';
+import { PropTypes } from 'react';
```

Comment:
Actually why do you not use https://www.npmjs.com/package/prop-types ? ...and why a second line for it?

Reply:
I don't think prop-types is available under our React version, and the reason I split it to a second line is because I know we'll need to change it to import from a different package in a later PR when we do the upgrade. ;)

Reply:
That package is completely independent of the underlying react version.

Reply:
First version of `prop-types` package is 15.5.0 and we're on 15.4.0. Even though React is good about compatibility, I don't think mixing and matching versions is a good idea. I don't think splitting an import into two lines is really a big deal is it? :)

Reply:
well I do not like having two lines and you are right they splitted the lib in 15.5 but that lib has no dep on any react, it is an independent util lib that you can even use in react from a year ago.